### ISSUE TITLE
Display css result directily for compile a single file via command line?

### ISSUE THREAD

**Issue by appleboy** [Compass/compass#1499]
Is there any way to prevent compass write output to file? Thanks.

ref issue: 

Let users handle their own IO nathggns/node-compass/issues/29
Can't process the output of the gulp-compass task. appleboy/gulp-compass/issues/2

**Comment by scottdavis** [comment]
so you are proposing that compass compile be able to pipe to stdout?

**Comment by scottdavis** [comment]
If thats the case it's probably not going to happen anytime soon that would take a lot of reengineering of the compiler.

**Comment by appleboy** [comment]
@scottdavis Yes, we need that compass compile be able to pipe to stdout, but the compass compile can't be able to display css result now. Right?

**Comment by scottdavis** [comment]
The compiler relies on an output file to check if it needs to compile. you can hack around this by creating your own sass engine and piping in the compass load paths but realize it will compile every time.

**Comment by nathggns** [comment]
I think the real issue for this would be `@import` statements.

**Comment by scottdavis** [comment]
you guys can always use `sass --compass`

**Comment by scottdavis** [comment]
https://github.com/nex3/sass/blob/79a145a7e30a70ae8361719eab4894e090be228a/lib/sass/exec.rb#L268
